content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -32,64 +27,33 @@ but not production. Like all alpha features, this guide is subject to change.
 
 ## Prepare the guide environment
 
-1. Set the environment variables `"${ISTIO_DIR}"`, `"${WORK_DIR}"`, your cluster
-    name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
+1. Create a virtual machine
+1. Set the environment variables `"${VM_NAME}"`, `"${WORK_DIR}"`,`"${CLUSTER_NAME}"` , `"${VM_NAMESPACE}"`,
+，`"${SERVICE_ACCOUNT}"` your cluster name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
     (e.g. `WORK_DIR="${HOME}"/vmintegration`).
 
     {{< text bash >}}
-    $ ISTIO_DIR="<the directory containing an unarchived version of Istio>"
+    $ VM_NAME="<the name of your vm instance you created>"
     $ CLUSTER_NAME="<the name of your cluster>"
```

Comment:
Cluster name serves no purpose here other than to make things more complicated. Instead of saving to `"${WORK_DIR}"/"${CLUSTER_NAME}"/"${VM_NAMESPACE}"` we can just save to `"${WORK_DIR}"`

Reply:
Ok